docs/source-app/workflows/byoc/index.rst

```diff
@@ -0,0 +1,99 @@
+
+#################################
+Run Apps on your own cloud (BYOC)
+#################################
+
+**Audience:** Users looking to run Lightning Apps on their own private cloud.
+
+----
+
+*******************
+A bit of background
+*******************
+
+BYOC - Bring Your Own Cloud, is an alternate deployment model to Lightning Cloud (fully managed SaaS).
+BYOC separates the control and data plane. The data plane, that includes
+Lightning clusters, services and Lightning Apps, reside inside the user’s VPC.
+The control plane resides on Lightning Cloud.
+
+Setup begins with configuring a cloud provider (today AWS, but more are coming soon) with your personal credentials for
+delegated access and an identity provider for secure access to the data plane.
+
+Next, as part of the environment creation process, you can configure networking,
+security, and select among cluster configuration options based on their own use cases.
+
+After submitting a cluster creation request, the Lightning Control Plane creates the required cloud infrastructure on the user account. This
+sets up a new Lightning Cluster along with a Lightning Kubernetes Operator.
+
+
+*******************************
+Create a Lightning BYOC cluster
+*******************************
+
+You must have your cloud configured before you try and create a BYOC cluster.
+
+Create a Lightning BYOC cluster using the following command:
+
+.. code:: python
+
+   lightning create cluster <cluster-name> <cloud-provider-parameters>
+
+Here's an example:
+
+.. code:: python
+
+   lightning create cluster my-byoc-cluster --provider aws --role-arn arn:aws:iam::1234567890:role/lai-byoc --external-id dummy --region us-west-2 --instance-types t3.xlarge --cost-savings
+
+..note:: Cluster creation is gonna take an hour or more after you run this command. 
+
+**Parameters:**
+
+* cluster_name: The name of the cluster to be created
+
+.. note:: Cluster names can only contain lowercase letters, numbers, and periodic hyphens ( - ).
+
+* provider: The cloud provider where your cluster is located.
+
+.. note:: AWS is supported today, but support for other cloud providers is coming soon.
+
+* role_arn: AWS IAM Role ARN used to provision resources
+
+* external_id: AWS IAM Role external ID
```

Comment:
based on wills feedback, consider linking to https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html to explain what an external ID is and why it's useful.

Reply:
Updated

Reply:
Updated